lib/Localization/LocalizationFormat.cpp

```diff
@@ -85,9 +91,24 @@ bool SerializedLocalizationWriter::emit(llvm::StringRef filePath) {
   return OS.has_error();
 }
 
+llvm::StringRef
+LocalizationProducer::getMessageOr(swift::DiagID id,
+                                   llvm::StringRef defaultMessage) {
+  auto localizedMessage = getMessage(id);
+  if (localizedMessage.empty())
+    return defaultMessage;
+  llvm::StringRef diagnosticName(diagnosticNameStrings[(unsigned)id]);
```

Comment:
I think this should be inside of `if (printDiagnosticNames)` because otherwise it's unused.